src/main/java/bhs/devilbotz/commands/assist/PickupFromGround.java

```diff
@@ -34,7 +34,8 @@ public PickupFromGround(Arm arm, Gripper gripper, DriveTrain drivetrain) {
     addCommands(
         new ParallelCommandGroup(
             // Drive backward
-            new SequentialCommandGroup(new DriveStraightPID(drivetrain, -.3), drivetrain.stop()),
+            new SequentialCommandGroup(
+                new DriveStraightPID(drivetrain, -.3), drivetrain.stopCommand()),
```

Comment:
```suggestion
                new DriveStraightPID(drivetrain, -0.3), drivetrain.stopCommand()),
```
Just for readability